### ISSUE TITLE
Remove leftovers from the Human Cell Atlas subdomain from the infrastructure playbook

### ISSUE THREAD

**Issue by domgz** [usegalaxy-eu/infrastructure-playbook#941]
Should not be needed anymore due to #939.

**Comment by bgruening** [comment]
Can we remove a subdomain from the DNS / AWS?

* remove website
* grep through all repos and remove stuff?

**Comment by kysrpex** [comment]
> Can we remove a subdomain from the DNS / AWS?
> 
>     * remove website
> 
>     * grep through all repos and remove stuff?

No, we should not remove the subdomain from AWS Route53 because the Human Cell Atlas subdomain needs to be redirected to the Single Cell subdomain forever. If we remove the DNS record then the redirect stops working.
